packages/ttpassgen/PKGBUILD

```diff
@@ -0,0 +1,29 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=ttpassgen
+_pkgname=TTPassGen
+pkgver=133.a06d99d
+pkgrel=1
+pkgdesc='highly flexible and scriptable password dictionary generator base on Python, you can easily use various rules to generate the desired combination of words.'
+arch=('any')
+groups=('blackarch' 'blackarch-automation' 'blackarch-cracker')
+url='https://github.com/tp7309/TTPassGen'
+license=('Apache')
+depends=('python' 'python-tqdm' 'python-click')
+makedepends=('git' 'python-setuptools')
+source=("git+https://github.com/tp7309/$_pkgname")
+sha512sums=('SKIP')
+
+build() {
+    cd "$_pkgname"
```

Comment:
Remove quotes.

Reply:
adjusting. Maybe update template on this as well ? Thanks.

Reply:
For some reasons, quotes are a good practice for some tools that have special characters or spaces in their name but if there's none of the mentioned above then quotes aren't needed.

Reply:
For example quote are needed when `cd $_pkgname-$pkgver"` because of the dash.